adserver/templates/adserver/advertisement.html

```diff
@@ -6,7 +6,7 @@
     <a href="{{ link_url }}" rel="nofollow" target="_blank">
       <img src="{{ image_url }}">
     </a>
-  {% elif view_url %}
+  {% elif view_url and publisher.render_pixel %}
```

Comment:
This is only the default template that isn't used. The real one is stored in the DB.

Reply:
Yea, I know -- was just an example and to keep things consistent.